llvm/tools/llvm-cas-object-format/llvm-cas-object-format.cpp

```diff
@@ -619,7 +634,19 @@ static void computeStats(CASDB &CAS, ArrayRef<CASID> TopLevels) {
   Collector.printToOuts(TopLevels);
 }
 
+static raw_fd_ostream &outCSV() {
+  std::error_code EC;
+  static raw_fd_ostream S(ComputeStats, EC, sys::fs::OF_None);
+  ExitOnError ExitOnErr;
+  ExitOnErr(errorCodeToError(EC));
+  return S;
+}
```

Comment:
I will be honest, I did this because that is how `outs() ` is implemented, and I didn't realize that the `raw_fd_ostream` it creates is static, thanks for pointing it out.

I didn't know of `Optional` so I can check it out.